setup.py

```diff
@@ -16,7 +16,13 @@ def readme():
         return f.read()
 
 
-INSTALL_REQUIRES = ["setuptools", "google-auth", "google-auth-oauthlib"]
+INSTALL_REQUIRES = [
+    "setuptools",
+    "google-auth >=1.25.0, <2.0dev;  python_version<'3.0'",
+    "google-auth; python_version>='3.6'",
+    "google-auth-oauthlib <0.4.2dev; python_version<'3.0'",
```

Comment:
0.4.1 was the last version to support Python 2.7 from what I can tell. A bit odd that 0.4.2 was a patch release. https://github.com/googleapis/google-auth-library-python-oauthlib/pull/93